extensions/api/util.py

```diff
@@ -95,4 +95,4 @@ def _start_cloudflared(port: int, max_attempts: int = 3, on_start: Optional[Call
             traceback.print_exc()
             time.sleep(3)
 
-        raise Exception('Could not start cloudflared.')
+        raise Exception('Could not start cloudflared.')
```

Comment:
There's some unrelated/unintended changes here that would be good to remove to make the PR easier to merge properly^

Reply:
I forgot to restore it, sorry